test/int/cluster_test.go

```diff
@@ -60,9 +58,14 @@ var _ = Describe("AtlasCluster", func() {
 			if createdCluster != nil {
 				By("Removing Atlas Cluster " + createdCluster.Name)
 				Expect(k8sClient.Delete(context.Background(), createdCluster)).To(Succeed())
-
 				Eventually(checkAtlasClusterRemoved(createdProject.Status.ID, createdCluster.Name), 600, interval).Should(BeTrue())
 			}
+
+			// TODO: CLOUDP-82115
```

Comment:
can we add TODO but leave the code here? Otherwise our projects won't be removed and we'll hit the Atlas limits (250 projects per organization)

Reply:
ah sorry dismiss this - I see the project is removed below 👍